README.md

```diff
@@ -195,6 +186,14 @@ We’d be really happy if you sent us links to your projects where you use our c
 - [BCReader](https://play.google.com/store/apps/details?id=com.iac.bcreader).
 - [Xprezia: Share Your Passion](https://play.google.com/store/apps/details?id=com.xprezzia.cnj).
 
+## Publishing a new version
+
+In the following cases, the CI will publish a new version with the following format to our remote Maven repo:
+
+* For each commit in an open PR: `<PR-number>-<commit full SHA1>`
+* Each time a PR is merged to `trunk`: `trunk-<commit full SHA1>`
```

Comment:
Are we renaming `develop` to `trunk`? I probably forgot to add one piece of information in the request - this repo has two important branches `develop` and `develop-non-native`. The one that we're interested in and planning to use is the second non-native one.

Reply:
Ah apologies @AdamGrzybkowski , this is a copy-pasting mistake, which I took from similar instructions we have on every other library of ours, I'll go ahead and rename `trunk` to `develop-non-native` on this `README.md` file.

> I probably forgot to add one piece of information in the request - this repo has two important branches develop and develop-non-native. The one that we're interested in and planning to use is the second non-native one.

Yea, I noticed that when I cloned the repo, seeing `develop-non-native` being the default branch, and so, I based everything on that, while ignoring `develop` for now. Thanks for the clarification! 🙇

Reply:
Also, now that we're talking about this, maybe I should delete this line as well, wdyt?

```
``` implementation 'com.automattic:ucrop:2.2.10-native' ``` - get power of the native code to preserve image quality (+ about 1.5 MB to an apk size)
```

And only keep this one line there:

```
``` implementation 'com.automattic:ucrop:2.2.10' ``` - lightweight general solution
```

Just to make it clear that we are only dealing/publishing the `non-native` side of things. 🤔

Reply:
Sounds good to me 👍

Reply:
This is now done: 404636a